pkg/flowaggregator/flowaggregator.go

```diff
@@ -222,10 +223,14 @@ func podInfoIndexFunc(obj interface{}) ([]string, error) {
 	if !ok {
 		return nil, fmt.Errorf("obj is not pod: %+v", obj)
 	}
-	if pod.Status.PodIP == "" {
-		klog.V(4).Infof("Pod %s/%s has not been assigned IP yet. ", pod.Namespace, pod.Name)
+	if pod.Status.PodIPs != nil && len(pod.Status.PodIPs) > 0 && pod.Status.Phase != corev1.PodSucceeded && pod.Status.Phase != corev1.PodFailed {
```

Comment:
I know this comes from the Traceflow controller, but the first 2 checks are redundant. `len(pod.Status.PodIPs)` will be 0 if `pod.Status.PodIPs` is nil